wallet-client/CLI.hs

```diff
@@ -0,0 +1,111 @@
+{-# LANGUAGE GADTs #-}
+
+module CLI
```

Comment:
Hmm, it would be nicer to reuse `app/CLI.hs` (move it to `cardano-node`, the library folder?) instead of duplicating parts here.

Reply:
+1, this will also hide some of the `ouroboros-consensus` dependencies.